src/libcadet/model/MultiChannelTransportModel.cpp

```diff
@@ -437,15 +448,23 @@ bool MultiChannelTransportModel::configure(IParameterProvider& paramProvider)
 		dynReactionConfSuccess = _dynReactionBulk->configure(paramProvider, _unitOpIdx, ParTypeIndep);
 		paramProvider.popScope();
 	}
+	
+	// Reconfigure exchange model
+	bool exchangeConfSuccess = true;
+	exchangeConfSuccess = _exchange[0]->configure(paramProvider, _unitOpIdx);
+	
+
 
-	return transportSuccess && dynReactionConfSuccess;
+	return transportSuccess && dynReactionConfSuccess && exchangeConfSuccess;
 }
 
 
 unsigned int MultiChannelTransportModel::threadLocalMemorySize() const CADET_NOEXCEPT
 {
 	LinearMemorySizer lms;
 
+	//Add exchange memory
```

Comment:
thats not actually done if i understand correctly?

Reply:
Do we need this? The size of the residuals does not change because we do not create a new state with the exchange module?

Reply:
no we currently dont need to do anything here, just delete the comment